api/src/gotEnzymes/crossReferencesMapping.js

```diff
@@ -51,4 +51,14 @@ export default {
     dbPrefix: 'reactome',
     compoundSuffix: '',
   },
+  ncbigene: {
+    db: 'NCBI Gene',
```

Comment:
This has to be:

```suggestion
    db: 'NCBI Protein',
```

Reply:
![image](https://user-images.githubusercontent.com/32480198/183434859-c1f06d9b-33c0-400c-b6f4-fec7376b8d52.png)

Reply:
This is how that would look, legit?

Reply:
Sorry @inghylt! committed just before I saw your message. I think we are on the same page though.

Reply:
Thanks for bringing this up @inghylt.

It's certainly confusing to have protein and gene next together but "NCBI Protein" is what it should say on the GotEnzymes page for genes. Something will need to be done about this identifier page, but this may be merged as is, and any feedback may be addressed separated.

In [another comment](https://github.com/MetabolicAtlas/MetabolicAtlas/pull/1062#discussion_r940920432) I'm proposing to forego the `referenceType` altogether, which would remove the word "gene" from this screenshot, and thus revolve the confusion.

I'll have a chat with @feiranl and @Yu-sysbio about this.